### PR TITLE
8 create profile component

### PR DESCRIPTION
**Pull request by DLiSWE**
Screenshot 2023-01-07 010228
Screenshot 2023-01-07 010101

### CODE REVIEW

File: src/features/theme/useTheme.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { toggleDarkMode, getCurrentTheme } from "features/theme/darkModeSlice";
 
 const useTheme = () => {
   const dispatch = useDispatch();
-  const theme = useSelector(getCurrentTheme);
+  const theme = !useSelector(getCurrentTheme);
```
Comment (raylu511) [inline src/features/theme/useTheme.tsx:6]: remove this

File: src/shared/components/ProfileCard/ProfileCard.tsx [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+        <CardContent
+          sx={{
+            bgcolor: theme.backgroundColor,
+            ...SharedFlexAbsoluteProps,
+            width: "100%",
+            top: "45%",
+            "@media (max-width:650px)": {
+              ...FlexAndCenterProps,
+              flexDirection: "column",
+              pt: "50px 0 unset",
```
Comment (raylu511) [inline src/shared/components/ProfileCard/ProfileCard.tsx:112]: i think this is supposed to be `p `and not `pt`

File: src/shared/components/ProfileCard/ProfileCard.tsx [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+                flexDirection: "column",
+                justifyContent: "flex-end",
+              }}
+            >
+              {isCurrentUser && (
+                <Button
+                  sx={{
+                    display: "flex",
+                    fontSize: "9px",
+                    py: "2px",
```
Comment (raylu511) [inline src/shared/components/ProfileCard/ProfileCard.tsx:245]: nit you can do `p: '2px 10px'`

File: src/shared/utils/profileUtils.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+export const getRandomColor = (string: string) => {
```
Comment (raylu511) [inline src/shared/utils/profileUtils.ts:1]: This is something I built a while back to get a randomColor that looks a bit cleaner
`export const getRandomColor = () => "#" + Math.floor(Math.random() * 16777215).toString(16)`

**Review by raylu511 — COMMENTED** [review]
Overall looks good. I would remove diffs that is unrelated to this ticket
I think just keep in mind that we should avoid doing more than whats expected for the ticket.

**Review by raylu511 — APPROVED** [review]

**Review by Matt-Tan15 — APPROVED** [review]

**Pull request merged**